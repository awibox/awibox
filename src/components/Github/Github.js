import React from 'react';
// Styles
import styles from './Github.scss';

const Github = () => {
  const hideRank = window.innerWidth < 415;
  return (
    <div className={styles.github}>
      <img className={styles.stats} src={`https://github-readme-stats.vercel.app/api?username=awibox&show_icons=true&hide_title=true&icon_color=fea621&title_color=fea621&hide_border=true&include_all_commits=true&count_private=true&hide_rank=${hideRank}`} alt='GitHub Stats' />
      <img className={styles.languages} src="https://github-readme-stats.vercel.app/api/top-langs/?username=awibox&layout=compact&langs_count=8&hide_border=true&hide_title=true" alt='Most used languages' />
    </div>
  );
};

export default Github;
