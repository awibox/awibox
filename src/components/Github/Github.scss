.github {
  margin-bottom: 25px;
  margin-top: -20px;
  margin-left: -25px;
  margin-right: -25px;
  justify-content: space-between;
  display: flex;
  flex-flow: wrap;
}

.stats {}

.languages {}

@media only screen and (min-width: 1455px) {
  .stats {
    max-width: 465px;
  }
  .languages {
    max-width: 335px;
  }
}
