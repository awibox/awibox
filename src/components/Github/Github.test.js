import React from 'react';
import { shallow } from 'enzyme';
import Github from './Github';

describe('Github', () => {
  it('should render correctly', () => {
    const component = shallow(<Github />);
    expect(component).toMatchSnapshot();
  });
});
