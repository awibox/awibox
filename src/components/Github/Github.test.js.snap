// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Github should render correctly 1`] = `
<div
  className="github"
>
  <img
    alt="GitHub Stats"
    className="stats"
    src="https://github-readme-stats.vercel.app/api?username=awibox&show_icons=true&hide_title=true&icon_color=fea621&title_color=fea621&hide_border=true&include_all_commits=true&count_private=true&hide_rank=false"
  />
  <img
    className="languages"
    src="https://github-readme-stats.vercel.app/api/top-langs/?username=awibox&layout=compact&langs_count=8&hide_border=true&hide_title=true"
  />
</div>
`;
